doc/da65.sgml

```diff
@@ -542,6 +542,18 @@ code. The following attributes are recognized:
   range, where <tt/label/ is the label name given with the <tt/NAME/
   attribute, and <tt/offs/ is the offset within the data.
 
+  <tag><tt>VOPERAND</tt></tag>
+  This optional attribute is followed by a numerical value. It tells the
+  assembler that subroutine calls to this label follow "virtual operands"
+  of the given bytes like this:
+
+<tscreen><verb>
+       	JSR	LabelWith2BytesOfVoperand
+	.byte	$00, $10    ; virtual operands
```

Comment:
As far as I understand your pull request doesn't make da65 ouput the "; virtual operands" so from my pov it should rather no be part of this sample, should it?